pkg/api/pool.go

```diff
@@ -34,6 +34,33 @@ type PoolPortal struct {
 	BasePortal
 }
 
+func (this *PoolPortal) ListAvailabilityZones() {
+	if !policy.Authorize(this.Ctx, "pool:get") {
```

Comment:
"pool:get" is an api of admin role. please change it to `"availabilityZone:list"`. Then, please update the policy.json in opensds/exmaples and opensds-installer/conf: `"availabilityZone:list":""`